moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -169,11 +169,6 @@ void RobotStateDisplay::changedEnableCollisionVisible()
   robot_->setCollisionVisible(enable_collision_visible_->getBool());
 }
 
-static bool operator!=(const std_msgs::msg::ColorRGBA& a, const std_msgs::msg::ColorRGBA& b)
```

Comment:
Why are you removing this operator?

Reply:
Its duplicated with the ColorRGBA message operator, which already contains the =! operator. So its not needed + throws a warning.